### ISSUE TITLE
not working in phonegap build

### ISSUE THREAD

**Issue by badpenguin** [appfeel/analytics-google#8]
If i add to my config.xml:
<gap:plugin name="com.analytics.google" />

then 
phonegap remote build android

will fail

**Comment by miqmago** [comment]
Yes, this is a problem of dependencies... Will see if can fix, normally it should be fixed with a new repo which uses google play plugin
